tests/datasource/fluent/test_spark_filesystem_datasource.py

```diff
@@ -11,6 +11,7 @@
 
```

Comment:
Open to any feedback on how to slim these tests down, but I think to test the splitter functionality we do need some data.